components/SiteNavbar.vue

```diff
@@ -1,56 +1,80 @@
 <template>
-  <nav class="border-b border-gray-300">
+  <div class="border-b border-gray-300">
     <div
       class="
         mx-auto
         container
-        flex flex-wrap flex-row-reverse
+        flex flex-wrap
         justify-between
         px-3
         py-3
         lg:flex-row
       "
     >
-      <div class="flex-row space-x-2 lg:hidden">
-        <!-- Mobile auth buttons -->
-        <a
-          class="
-            rounded
-            px-3
-            py-2
-            border border-gray-300
-            text-gray-600
-            hover:text-gray-800 hover:bg-gray-100
-          "
-          href="#"
-          >Log In</a
-        >
-        <a
-          class="
-            bg-gray-600
-            rounded
-            px-3
-            py-2
-            text-gray-100
-            hover:text-white hover:bg-gray-900
-          "
-          href="#"
-          >Sign Up</a
-        >
+      <NuxtLink class="flex flex-grow" to="/">
+        <div class="flex">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            class="h-10 w-10 mx-1"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="M12 8v4l3 3m6-3a9 9 0 11-18 0 9 9 0 0118 0z"
+            />
+          </svg>
+          <h1
+            class="
+              w-full
+              py-2
+              text-lg text-center md:text-left
+              font-semibold
+              leading-6
+              tracking-tight
+              hidden lg:inline
+            "
+          >
+            speedrun.website
+          </h1>
+        </div>
+      </NuxtLink>
+      <div
+        v-show="lg || mobileNavIsActive"
+        class="
+          border-t border-gray-300 lg:border-t-0
+          flex flex-col lg:flex-row
+          mt-2 lg:mt-0
+          w-full lg:w-auto
+          space-y-2 lg:space-y-0
+        "
+      >
+        <CoreNavLinks />
+        <CoreSearchBar />
+      </div>
+      <div class="
+             flex flex-row
+             absolute right-2 top-4
+             lg:relative lg:right-0 lg:top-0
+             space-x-2 lg:space-x-0
```

Comment:
could you add the `items-center` class to this. Right now the child buttons height is being stretched to the height of this element making them taller than the other buttons. This will centre the buttons vertically and keep their heights consistent with the others.